Anlab.Mvc/Services/IOrderService.cs

```diff
@@ -176,7 +176,8 @@ private static void CalculateTest(OrderDetails orderDetails, TestItemModel test,
             orderDetails.Total = orderDetails.SelectedTests.Sum(x=>x.Total);
 
             orderToUpdate.SaveDetails(orderDetails);
-            orderToUpdate.AdditionalEmails = string.Join(";", orderDetails.AdditionalEmails);
+
+            orderToUpdate.AdditionalEmails = orderDetails.AdditionalEmails != null ? string.Join(";", orderDetails.AdditionalEmails): null;

```

Comment:
@jSylvestre another point where if you just defaulted to an empty array you wouldn't need the check.